src/components/common/StaffPayment.vue

```diff
@@ -127,4 +127,7 @@ export default class StaffPayment extends Vue {
   color: $gray9;
 }
 
+::v-deep #staff-payment-container {
+  padding-left: 1rem;
```

Comment:
Why wouldn't this padding be in THIS component?

Reply:
(This component wraps the staff payment sub-component. The sub-components should have no extra whitespace, so the parent component can specify it.)
I mean, your code works, but why reach into the component to increase its whitespace instead of just specifying the whitespace here. This might actually work:
```
<StaffPayment class="pl-4" ...
```

Reply:
To all: as a general rule, we should NOT reach into components (with v-deep) to set their styling. It's hacky and fragile.